### DISCUSSION TITLE
After built, how can I run rustdesk app in MacOS just click "rustdesk exec file" but by CLI "target/release/rustdesk" ?

### DISCUSSION THREAD

**Discussion by kvsur** [rustdesk/rustdesk#5619] [Q&A]
I had pull source code and build it use `cargo build --release`, but when I click exec file it now working, I just want run app by this way rather than use CLI command, can anyone give me help?
<img width="570" alt="image" src="https://github.com/rustdesk/rustdesk/assets/13591479/f37ac6e4-f370-4a39-bdc8-d4ac4ed16d0d">

**Comment by SergeyMy** [comment]
copy the entire folder from the source folder
src/ui to the plcopy the entire folder from the source folder
![изображение](https://github.com/rustdesk/rustdesk/assets/131688106/c4b27d7c-667a-44d5-87df-7045f2332779)
PS sciter version is outdated, you need to collect flutter
